lua/harpoon/mark.lua

```diff
@@ -178,16 +194,15 @@ M.set_mark_list = function(new_list)
 end
 
 M.toggle_file = function(file_name_or_buf_id)
-    local mark_count_before = #harpoon.get_mark_config().marks
-
-    M.add_file(file_name_or_buf_id)
+    local buf_name = get_buf_name(file_name_or_buf_id)
 
-    local mark_count_after = #harpoon.get_mark_config().marks
+    validate_buf_name(buf_name)
```

Comment:
What's the point of this?  It has no early return, it will simply just print an error message.

Reply:
I believe `error` raises an error and the code stops execution (unless the error is handled). If we were using `print`, the execution would continue.